src/main/java/sparksoniq/jsoniq/runtime/iterator/RuntimeIterator.java

```diff
@@ -45,12 +45,15 @@
 
 	private static final long serialVersionUID = 1L;
 	protected static final String FLOW_EXCEPTION_MESSAGE = "Invalid next() call; ";
-    private final IteratorMetadata metadata;
+    private IteratorMetadata metadata;
     protected boolean _hasNext;
     protected boolean _isOpen;
     protected List<RuntimeIterator> _children;
```

Comment:
This is the "meat", together with the IteratorMetadata, the only thing that needs to be serialized (i.e., not transient) here.